src/components/views/rooms/RoomSettings.js

```diff
@@ -652,8 +655,9 @@ module.exports = React.createClass({
                     <ul className="mx_RoomSettings_banned">
                         {banned.map(function(member) {
                             const banEvent = member.events.member.getContent();
+                            const bannedBy = self.props.room.getMember(member.events.member.getSender());
```

Comment:
This could potentially be `null` but `BannedUser` has a requirement for `by`. I suggest falling back to the user ID for the sender if `by` is falsey.